src/Icon/consts.js

```diff
@@ -11,5 +11,5 @@ export const ICON_COLORS = {
   attention: defaultTokens.orbit.colorIconAttention,
   primary: defaultTokens.orbit.colorIconPrimary,
   secondary: defaultTokens.orbit.colorIconSecondary,
-  terciary: defaultTokens.orbit.colorIconTerciary,
+  tertiary: defaultTokens.orbit.colorIconTerciary,
```

Comment:
we should fix also tokens naming

Reply:
Cannot right know, I have it in my TODOs for new tokens version.